### PR TITLE
Hero banner linearizer

### LINKED ISSUE #1543: [Home] Hero: Create section randomizer
**Current Behavior**
Different versions of the hero section are available. The hero section is statically configured, though.

**Desired Behavior**
Ideally, the hero section is set to randomly load a design from a collection of designs.

This issue calls for implementation of a hero randomizer akin to the randomizer on the 404 error page.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by vineethvanga18**
Signed-off-by: vineethvanga18 <reddy.8@iitj.ac.in>

**Description**
Added hero banner linearizer

This PR fixes #1543

**Notes for Reviewers**
1. Create a banner directory in 'sections/Home' directory, like Banner-default, Banner-Alt.
1. Import that banner and add it to Banners array in 'sections/Home/Banner/index.js'. 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Home/Banner/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import BannerDefault from "../Banner-default";
+// import BannerAlt from "../Banner-alt"; 
+
+const Banners = [
+  <BannerDefault key={0}/>,
+  // <BannerAlt key={1}/> 
+];
+
+const Banner = () => {
+  const [Banner, setBanner] = useState(sessionStorage.getItem("banner") || 0);
```
Comment (EmperorYP7) [inline src/sections/Home/Banner/index.js:11]: shouldn't it be `window.sessionStorage`?

**Review by EmperorYP7 — CHANGES_REQUESTED** [review]

File: src/sections/Home/Banner/index.js (hunk 1 above)
Comment (vineethvanga18) [reply src/sections/Home/Banner/index.js:11]: sessionStorage refers to window.sessionStorage. We can simply use sessionStorage if no other variable of the same name exists.

Comment (EmperorYP7) [reply src/sections/Home/Banner/index.js]: Yea, you're right. Thanks! :smile:

**Comment by leecalcote** [comment]
@EmperorYP7 are you good to go in term of refactoring your PR to fit into this rotation scheme?

**Review by EmperorYP7 — APPROVED** [review]

**Comment by EmperorYP7** [comment]
> @EmperorYP7 are you good to go in term of refactoring your PR to fit into this rotation scheme?

Yes!

**Review by leecalcote — CHANGES_REQUESTED** [review]
I get a blank screen when checking out and building this branch.

**Comment by vineethvanga18** [comment]
@leecalcote Do you see any error in the console?

**Comment by leecalcote** [comment]
[localhost-1614625917588.log](https://github.com/layer5io/layer5/files/6063927/localhost-1614625917588.log)
@vineethvanga18 yes, here's a copy of the error.

**Comment by vineethvanga18** [comment]
@leecalcote could you try it now?

**Comment by leecalcote** [comment]
Yes, it builds and displays successfully now. Merging so that the banner-alt work can continue...

**Review by leecalcote — APPROVED** [review]

**Comment by vineethvanga18** [comment]
@leecalcote I should have labelled it as onhold

**Pull request merged**